### PR TITLE
Feat/display ppcs on map

### PR DESCRIPTION
**Pull request by joaquimds**
## Description
Adds PPCs to the hub map. Also improves UX when loading the map on a postcode by preventing the fly out and fly back in. Also attempts to improve the map height, and makes the panel scrollable.

## Motivation and Context
https://linear.app/commonknowledge/issue/CC-84/display-some-lih-data-including-ppc

## How Can It Be Tested?
Import the PPCs:

`python manage.py import_2024_ppcs`

Then search for a postcode on the hub map.

## How Will This Be Deployed?
Normal deployment, need to run the import command afterwards.

### CODE REVIEW

File: nextjs/src/app/hub/render/[hostname]/map/[[...postcodes]]/page.tsx [hunk 1]
```diff
@@ -146,6 +142,25 @@ const GET_LOCAL_DATA = gql`
...
+          }) {
+            name: data
+            shade
+          }
+          email: personDatum(filters:{
+            dataType_Name: "email"
+          }) {
+            data
+          }
+        }
```
Comment (janbaykara) [inline nextjs/src/app/hub/render/[hostname]/map/[[...postcodes]]/page.tsx:163]: Ah I love it when an API does the job.

**Comment by conatus** [comment]
To keep this CSV fresh, perhaps we should have a cron that grabs the CSV, say, every hour or so.

https://candidates.democracyclub.org.uk/data/?election_date=&ballot_paper_id=&election_id=parl.2024-07-04&party_id=&cancelled=&field_group=election&field_group=person

**Comment by conatus** [comment]
I stand corrected, the script already does a remove fetch for the data, so we could just set it to run. Lovely stuff @joaquimds!

**Comment by janbaykara** [comment]
Have made a task to run a cron

**Pull request merged**